packages/assets-controllers/src/TokensController.ts

```diff
@@ -960,6 +961,22 @@ export class TokensController extends BaseController<
       );
     }
   }
+
+  initApprovals() {
```

Comment:
Also, as this is public, we probably want a comment, return type, and to move it above the other private methods.

Reply:
I moved it to the bottom of the group that handles the watch token feature.